recipes/glibc/glibc.inc

```diff
@@ -43,6 +43,7 @@ do_configure_siteinfo() {
 }
 
 EXTRA_OECONF += "\
+	libc_cv_slibdir='/lib' \
```

Comment:
Yes, so you say :)  But I am still waiting for instructions on how to reproduce this.  It seems strange that such a fundamental problem has been unseen for such a long time.

I still suspect that it is caused by the output from "$CC -print-multi-os-directory" of the native compiler on specific Linux distros.  Maybe we need to ignore -print-multi-os-directory for native compilers, but I don't think it should be ignored for cross and sdk-cross compilers that we build ourselves.